Vagrantfile

```diff
@@ -48,22 +48,29 @@ echo "Launching bootstrap"
 exec "/srv/scality/metalk8s-$SHORT_VERSION/bootstrap.sh"
 SCRIPT
 
+NETWORK_RANGE = "192.168.42."
+BOOTSTRAP_IP_OFFSET = 10
+
 # To support VirtualBox linked clones
 Vagrant.require_version(">= 1.8")
 
 Vagrant.configure("2") do |config|
-  config.vm.define :bootstrap do |bootstrap|
-    bootstrap.vm.box = "centos/7"
-    bootstrap.vm.box_version = "1811.02"
+  config.vm.box = "centos/7"
+  config.vm.box_version = "1811.02"
+
+  config.vm.provider "virtualbox" do |v|
+    v.linked_clone = true
+    v.memory = 2048
```

Comment:
This (and CPU) is not necessarily the same across all nodes. As an example, I bump the `bootstrap` node memory to 4GB. However, other control-plane nodes could stay at 2GB (for now, may need some more memory once/if we deploy ES), and workload-plane nodes at 1GB, or even lower.

Reply:
Yeah, I'll wait for your PR with the 4GB RAM to be merged to integrate this new value.
I'm not sure we want to have all kind of nodes in the Vagrant environment, at least right now, and since it would be difficult to have a lot of those nodes running at the same time, we may not be able to reproduce any kind of environment though.

Reply:
Let's wait to have a clear vision on which kind of environments we want before setting different values for nodes.